api/src/main/java/javax/enterprise/inject/spi/CDIProvider.java

```diff
@@ -15,4 +18,31 @@
      */
     public CDI<Object> getCDI();
 
+    /**
+     * Determines whether or not this CDIProvider has been initialized or not.
+     * @return true if initialized, false if not.
+     */
+    boolean isInitialized();
+
+    /**
+     * Initializes a CDI Container.  Cannot be called within an application server or if the container is already started.
```

Comment:
Update the javadoc to allow multiple call like you allowed it in the spec.

````
Initializes a CDI Container.  If called within an application server, an {@link java.lang.IllegalStateException} is thrown.
`````